api/package.json

```diff
@@ -34,6 +34,7 @@
     "create-migration": "ts-node ./migrations/config/create.ts"
   },
   "dependencies": {
+    "@google/generative-ai": "^0.19.0",
```

Comment:
Thank you @medchedli for your contribution 🥳 

Let's rather create a package.json under src/extensions/plugins/gemini/

In which we would put the dependency. Check the way OpenAI Chatgpt plugin is done 🙏